.github/workflows/CI.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - "1.0"  # LTS
+          - "1.3"  # Oldest release supported by Arrow.jl
```

Comment:
Need to either set the package Julia company to 1.3 or we should add `VERSION` checks and continue to support Julia 1.0 but disable Arrow integration

Reply:
The thing is that this package will continue working on 1.0 and we can't use version checks because the test dependencies will just fail to install before we even try to load our code.

Reply:
Ah, I see. I dislike not being able to test on the older version supported by the package but I understand the change now.